packages/docusaurus-theme-openapi/src/theme/ApiItem/index.tsx

```diff
@@ -28,7 +28,7 @@ function ApiItem(props: Props): JSX.Element {
 
   return (
     <>
-      <Seo {...{ title, description, keywords, image }} />
+      {/* <Seo {...{ title, description, keywords, image }} /> */}
```

Comment:
Couldn't find a replacement but I'm sure it's there somewhere

Reply:
I think this was replaced with a `PageMetadata` component.

```
import { PageMetadata } from "@docusaurus/theme-common";
```

Reply:
fixed